### DISCUSSION TITLE
Hello Pls Help Meee

### DISCUSSION THREAD

**Discussion by Zhask51** [termux/termux-packages#7341] [General]
if im save/copy/move on termux mc it will show a error

im trying to save a file but

(mc:31527): GLib-ERROR **: 11:38:39.096: ../src/glib/gmem.c:112: failed to allocate 18446744073709551560 bytes
                                                   [1]    31527 trace trap  mc
                   %

help me

**Comment by ghost** [comment]
This issue has been reported in https://github.com/termux/termux-packages/issues/7324.

Someone will need to debug and patch it. Allocating 18446744073709551560 bytes of memory is too silly (no devices have so much memory). Maybe bug introduced with update.
